activerecord/lib/active_record/connection_adapters/abstract/quoting.rb

```diff
@@ -43,6 +43,10 @@ def type_cast(value, column = nil)
         _type_cast(value)
       end
 
+      def quote_bound_value(value)
```

Comment:
Either `# :nodoc:` or document it if it is intended to be public API.

Reply:
I'll document it, as I think some custom adapters might want to define it.